app/templates/suppliers/edit_supplier_trading_status.html

```diff
@@ -31,12 +33,27 @@
 <div class="single-question-page">
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-l">What&rsquo;s your trading status?</h1>
-
       <form method="POST" action="{{ url_for('.edit_supplier_trading_status') }}">
```

Comment:
We should add `novalidate` to forms to avoid issues with browser-specific validation. https://design-system.service.gov.uk/patterns/validation/#turn-off-html5-validation